.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
I can't recall the exact scenario, but we ended up making this more liberal in our current app, >.10% and removed the last 2 versions constraint. Curious your thoughts on this

Reply:
i wish I had taken better notes on what exactly caused this, I think it was related to me not updating firefox for a few months and a new function not being supported. I know we had it set to 1% for a year or so it was fine, so we could always take the more strict approach until something breaks.

Reply:
As part of habit, I won't be using any javascript or css features that aren't supported by all modern browsers (everything but IE). So this shouldn't be very relevant.

Also re: `last 2 versions`, I want to keep an attitude of forward-looking/modern, and also avoid situations where a really old version still has support, like in the case of IE9 having `0.12%` for some unholy reason.